src/components/calcite-radio-button/calcite-radio-button.tsx

```diff
@@ -122,16 +123,18 @@ export class CalciteRadioButton {
   //--------------------------------------------------------------------------
 
   private checkFirstRadioButton(): void {
-    const radioButtons = document.querySelectorAll(`calcite-radio-button[name=${this.name}]`);
+    const radioButtons = Array.from(document.querySelectorAll("calcite-radio-button")).filter(
+      (radioButton) => radioButton.name === this.name
+    ) as HTMLCalciteRadioButtonElement[];
```

Comment:
For some reason the `name` attribute is not reflected properly in the DOM, so this querySelector needed to be updated to check for the property instead.  Not sure what changed, but this smells like a Stencil regression to me.